### PR TITLE
Email model and service

### PR DESCRIPTION
**Pull request by brettminnie**
Do not merge

### CODE REVIEW

File: models/delivery.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var deliveryModel = require('hmpo-model');
```
Comment (JoeChapman) [inline models/delivery.js]: look at this: https://github.com/UKHomeOffice/passports-model
when you add to the prototype of a constructor, you are essentially defining new methods on any new instance of that constructor/class. So if someone else `new`'s up that constructor/class they'll share any new prototype methods you've added

Comment (brettminnie) [inline models/delivery.js]: Aah, I was viewing it as more traditional inheritance vs partials... I'll
read up some

Thanks

Comment (lennym) [inline models/delivery.js]: If you want to extend the Model classs with additional prototype methods:

```javascript
var util = require('util');

function DeliveryModel() {
  Model.call(this);
}
util.inherits(DeliveryModel, Model);

DeliveryModel.prototype.foo = function () {
...
};

module.exports = DeliveryModel;
```

**Comment by easternbloc** [comment]
Closing this for now until we figure out the email solution.

**Comment by brettminnie** [comment]
I should have deleted this, should be superseded by the email app prototype
I knocked up

**Comment by easternbloc** [comment]
@brettminnie yeah probably but I'm gonna leave it here till we implement the email service. Cheers dude.

**Comment by brettminnie** [comment]
:+1:

Pity you can't convert a pull request into an issue :)

**Pull request closed without merging**